File: (7kyu) Form The Largest/(7kyu) Form The Largest.js
// #1
// function maxNumber(n) {
//   const str = String(n);
//   const arr = str.split('');
//   const arrS = arr.sort((a, b) => b - a);
//   const res = arrS.join('');
//   return parseInt(res);
// }
// #2
// function maxNumber(n) {
//   return parseInt(
//     String(n)
//       .split('')
//       .sort((a, b) => b - a)
//       .join(''),
//   );
// }
// #3
// const maxNumber = (n) =>
//   parseInt(
//     String(n)
//       .split('')
//       .sort((a, b) => b - a)
//       .join(''),
//   );
// #4
// const maxNumber = (n) => Number(String(n).split``.sort().reverse().join``);
// #5
const maxNumber = (n) => +String(n).split``.sort().reverse().join``;

File: (7kyu) Form The Largest/(7kyu) Form The Largest.py
# #1
# def max_number(n):
#     s = list(str(n))
#     s.sort()
#     res = s[::-1]
#     return int("".join(res))
# #2
# def max_number(n):
#     s = list(str(n))    
#     return int(''.join(sorted(s, reverse=True)))
#3
def max_number(n):
    return int(''.join(sorted(str(n), reverse=True)))
